src/tox/_pytestplugin.py

```diff
@@ -69,7 +69,7 @@ def cmd(request, capfd, monkeypatch):
     request.addfinalizer(py.path.local().chdir)
 
     def run(*argv):
-        key = str(b"PYTHONPATH")
+        key = b"PYTHONPATH" if six.PY2 else "PYTHONPATH"
```

Comment:
"native" string can just be `str("PYTHONPATH")` -- though as written is just as explicit and correct